### DISCUSSION TITLE
What's going on with Documentation?

### DISCUSSION THREAD

**Discussion by BubbaRichard** [MonoGame/MonoGame#8148] [General]
I went to the documentation on MonoGame.org and searched for a few different items like DrawIndexedPrimitives and they have disappeared. I know they used to bring up results but now nothing. Is this just a migration thing?

Thanks,
BubbaRichard

**Comment by ThomasFOG** [comment]
Here it is: https://monogame.net/api/Microsoft.Xna.Framework.Graphics.GraphicsDevice/#drawindexedprimitives-primitivetype-int-int-int-int-int

We're currently in the process of migrating the URLs from ```docs.monogame.net``` to ```monogame.net``` and a redirection will be made.

**Comment by AristurtleDev** [comment]
This should be resolved once https://github.com/MonoGame/monogame.github.io/pull/91 is pushed through
